qgrad/qutip.py

```diff
@@ -0,0 +1,45 @@
+"""
+QuTiP functions that work with Jax
+"""
+
+from jax import numpy as jnp
```

Comment:
I think it is commonplace to write `import jax.numpy as jnp`, isn't it?

Reply:
Both are fine but I like yours because it is shorter. Will update.